lib/benefits_intake_service/service.rb

```diff
@@ -52,6 +52,20 @@ def get_upload_location
       perform :post, 'uploads', request_body, headers
     end
 
+    def get_bulk_status_of_uploads(ids)
+      body = { ids: }.to_json
+      response = perform(
+        :post,
+        'uploads/report',
+        body,
+        { 'Content-Type' => 'application/json', 'accept' => 'application/json' }
+      )
+
+      raise response.body unless response.success?
+
+      response
+    end
```

Comment:
I didn't write a test for this because I think I didn't see tests for other methods in this file and there's not really any logic in here, per se.